blog/_posts/2018-06-10-bsp.md

```diff
@@ -0,0 +1,214 @@
+from Bloop via BSP. In total, the "Import Project" process takes around ~15
+seconds.
+
+You can see it on this demo importing [akka/akka](https://github.com/akka/akka/):
+
+![Demo of BSP import for akka/akka](/resources/img/bloop-install.gif)
+
+In my experience, importing a build via BSP is substantially faster than
+doing so via the traditional sbt or Maven project import where the process
+can take minutes. It also enables smarter communication between build tools
+and IDEs because the communication is bidirectional.
+
+Further, this two-step process can be improved in the future. First, bloop
+could run `bloopInstall` every time you start up your sbt shell or `reload`
+your build. Second, Bloop's Build Server could send notifications the IDE
+whenever there is a change in the build. These enhancements would improve the
+state-of-the-art in IDE project import by not requiring users to manually
+check when the build and IDE have gone off-sync and ask for a "Reload
+Project". If sbt implemented BSP, an sbt shell would suffice to also support
+a project import with these properties instantly.
+
+Aside from importing your build, the BSP prototype also allows you to get
+compiler diagnostics directly from your build tool instead of diagnostics
+from IntelliJ's presentation compiler. Build tool diagnostics are always
+correct since the compilation happens in your build tool and the results
+are streamed to the IDE.
+
+![Demo of BSP compile for akka/akka](/resources/img/bloop-compile.gif)
+
+The BSP integration is now available under an experimental mode in the latest
+IntelliJ 2018.2 nightly. To try the prototype out, follow the instructions in
+[this
+document](https://gist.github.com/jvican/0d738cc0aa47afe5dbbdfe6ea1bd8694).
+
+To learn more about the Build Server Protocol, check out the
+[slides](http://jorge.vican.me/slides/BSP.pdf) of my talk with Justin at
+Scalasphere 2018.
+
+## Next steps
+
+The next steps will focus on two main areas: adoption and user impact. First,
+in order to ease support of all build tools in IntelliJ, the Scala Center
+will keep collaborating with Jetbrains to improve IntelliJ's BSP import for
+Bloop and other build tools. The goal is to bring BSP import to a
+production-ready level where users can benefit from a build-tool-agnostic
+import project that is both faster and more reliable.
+
+Second, there is work to do to create a better synergy between build tool
+authors and IDE implementors. The [Scala Tooling Protocol Working Group]()
+that the Scala Center created a few months ago will help us discuss and
+refine the protocol so that all tooling developers in our community can
+implement the protocol at scale.
+
+Third, I'd like to collaborate more closely with developers working on Scala
+language servers like [Metals] or [Dotty IDE] to bring Scala support to all
+editors. There is a big chunk of Scala developers that don't use IntelliJ and
```

Comment:
There's a singular/plural mismatch between `is`, `chunk` and `developers`. You probably shouldn't use `chunk` on a plural thing, and it's kind of the wrong word to use to describe people... How about just `many Scala developers"?

Also, s/that/who/